Yubico.YubiKey/src/Yubico/YubiKey/Piv/PivSession.ManagementKey.cs

```diff
@@ -71,6 +71,12 @@ public sealed partial class PivSession : IDisposable
         /// </remarks>
         public AuthenticateManagementKeyResult ManagementKeyAuthenticationResult { get; private set; }
 
+        private PivAlgorithm DefaultManagementKeyAlgorithm =>
+            _yubiKeyDevice.HasFeature(YubiKeyFeature.PivAesManagementKey) &&
+            _yubiKeyDevice.FirmwareVersion > FirmwareVersion.V5_7_0
```

Comment:
The default AES192 management key was introduced in FW 5.7.0, I think the condition on this line should be `>=`